presto-native-tests/src/test/java/com/facebook/presto/nativetests/NativeTestsUtils.java

```diff
@@ -47,10 +47,10 @@ public static void createTables(String storageFormat)
                     .setAddStorageFormatToPath(true)
                     .build();
             if (storageFormat.equals("DWRF")) {
-                NativeQueryRunnerUtils.createAllTables(javaQueryRunner, true);
+                NativeQueryRunnerUtils.createAllTablesStandard(javaQueryRunner, true);
```

Comment:
Don't understand why you changed this to standard.. Seems non-intuitive as the calling method is called createTables.